app/models/spree/variant_decorator.rb

```diff
@@ -8,7 +8,7 @@ module Spree
 
     private
 
-    durably_decorate :create_stock_items, mode: 'soft', sha: '4f82bc9c291bbef359b490e622b2b2c58d9691ee' do
+    durably_decorate :create_stock_items, mode: 'soft', sha: '98704433ac5c66ba46e02699f3cf03d13d4f1281' do
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.